docs/specs/escape.yml

```diff
@@ -39,28 +41,32 @@ outputs:
     }
   'a b.json': |
     {
-      "canonical_url": "https://docs.com/en-us/a b",
-      "gitcommit": "https://github.com/escape/markdown/blob/c4c07cb69d114fa9ba16e55763d47f2344083dad/a b.md",
+      "canonical_url": "https://docs.com/en-us/a%20b",
+      "gitcommit": "https://github.com/escape/markdown/blob/03cb6e2fa19c69303dd589df63df0c40448b241b/a b.md",
       "content_git_url": "https://github.com/escape/markdown/blob/main/a b.md",
       "original_content_git_url": "https://github.com/escape/markdown/blob/main/a b.md",
       "original_content_git_url_template": "{repo}/blob/{branch}/a b.md"
     }
   a&b.json: |
     {
       "canonical_url": "https://docs.com/en-us/a&b",
-      "gitcommit": "https://github.com/escape/markdown/blob/c4c07cb69d114fa9ba16e55763d47f2344083dad/a&b.md",
+      "gitcommit": "https://github.com/escape/markdown/blob/03cb6e2fa19c69303dd589df63df0c40448b241b/a&b.md",
       "content_git_url": "https://github.com/escape/markdown/blob/main/a&b.md",
       "original_content_git_url": "https://github.com/escape/markdown/blob/main/a&b.md",
       "original_content_git_url_template": "{repo}/blob/{branch}/a&b.md"
     }
   a&amp;b.json: |
     {
       "canonical_url": "https://docs.com/en-us/a&amp;b",
-      "gitcommit": "https://github.com/escape/markdown/blob/c4c07cb69d114fa9ba16e55763d47f2344083dad/a&amp;b.md",
+      "gitcommit": "https://github.com/escape/markdown/blob/03cb6e2fa19c69303dd589df63df0c40448b241b/a&amp;b.md",
       "content_git_url": "https://github.com/escape/markdown/blob/main/a&amp;b.md",
       "original_content_git_url": "https://github.com/escape/markdown/blob/main/a&amp;b.md",
       "original_content_git_url_template": "{repo}/blob/{branch}/a&amp;b.md"
     }
+  a – b c/ab.json: | 
```

Comment:
Do we know why `-` is encoded? I think it is a valid char for URL per [RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986).